lib/mpool/include/mpool/mpool_structs.h

```diff
@@ -26,9 +26,17 @@
 #define MPOOL_CAPACITY_MCLASS_DEFAULT_PATH "capacity"
 #define MPOOL_PMEM_MCLASS_DEFAULT_PATH     "pmem"
 
+#define MPOOL_MBLOCK_SIZE_MIN          (1ul << MB_SHIFT)
+#define MPOOL_MBLOCK_SIZE_MAX          (1024ul << MB_SHIFT)
 #define MPOOL_MBLOCK_SIZE_DEFAULT      (32ul << MB_SHIFT)
-#define MPOOL_MBLOCK_FILECNT_DEFAULT   (32)
-#define MPOOL_MBLOCK_FILESZ_DEFAULT    (2048ull << GB_SHIFT)
+
+#define MPOOL_MCLASS_FILECNT_MIN       (1)
```

Comment:
Can you move these to an mpool/limits.h header? This helps move the code organization toward a vision that I have.

Reply:
Done.